### PR TITLE
Localization Support for Options

### PR DESCRIPTION
**Pull request by IBRS-4Ever**
image

I couldn't find the file that defines the text 'Legal'.

Moved 
"GameUI_HTMLLoadingScreen" "Enable custom HTML loading screens"
"GameUI_CrosshairCustomize" "Crosshair Customization"
From garrysmod_english.txt to gameui_english.txt

image

### CODE REVIEW

File: garrysmod/resource/OptionsSubAudio.res [hunk 1]
```diff
@@ -471,7 +471,7 @@
 		"visible"		"1"
 		"enabled"		"1"
 		"tabPosition"		"0"
-		"labelText"		"Sound effect volume"
+		"labelText"		"#GameUI_SoundEffectVolume"
```
Comment (robotboy655) [inline garrysmod/resource/OptionsSubAudio.res:474]: You cannot not change existing localization strings.

`#GameUI_SoundEffectVolume` should remain where it was, and a new string can be added for `Sound effect volume` (for example `"#GameUI_SoundFXVolume"`)

File: garrysmod/resource/OptionsSubGModLegal.res [hunk 2]
```diff
@@ -153,7 +153,7 @@
 		"visible"		"1"
 		"enabled"		"1"
 		"tabPosition"		"4"
-		"labelText"		"Chromium Embedded Framework Credits"
+		"labelText"		"#GarrysMod_Legal_ChromiumEmbeddedFrameworkCredits"
```
Comment (robotboy655) [inline garrysmod/resource/OptionsSubGModLegal.res:156]: the string could be shortened for `#GarrysMod_Legal_CEFCredits`

**Review by robotboy655 — CHANGES_REQUESTED** [review]
I will change the `Legal` tab to use `#GameUI_Legal` or something similar after you fix the PR and it is merged.

**Comment by IBRS-4Ever** [comment]
Fixed:
image

**Review by robotboy655 — APPROVED** [review]

**Pull request merged**